packages_rs/nextclade/src/align/seed_alignment.rs

```diff
@@ -143,186 +143,190 @@ pub fn seed_alignment(
   seed_index: &CodonSpacedIndex,
   params: &AlignPairwiseParams,
 ) -> Result<Vec<Stripe>, Report> {
-  let qry_len_u = qry_seq.len();
-  let ref_len_u = ref_seq.len();
-  let qry_len_i = qry_len_u as i32;
-  let ref_len_i = ref_len_u as i32;
+  let qry_len = qry_seq.len();
+  let ref_len = ref_seq.len();
 
-  if ref_len_u + qry_len_u < (5 * params.seed_length) {
+  if ref_len + qry_len < (10 * params.seed_length) {
     // for very short sequences, use full square
-    let stripes = full_matrix(ref_len_u, qry_len_u);
+    let stripes = full_matrix(ref_len, qry_len);
     trace!("Band construction: Short qry&ref sequence (< 5*seed_length), thus using full matrix");
     Ok(stripes)
   } else {
     // otherwise, determine seed matches roughly regularly spaced along the query sequence
     let seed_matches = get_seed_matches2(qry_seq, ref_seq, seed_index, params)?;
     create_stripes(
       &seed_matches,
-      qry_len_i,
-      ref_len_i,
-      params.terminal_bandwidth,
-      params.excess_bandwidth,
-      params.max_indel,
-      params.allowed_mismatches,
+      qry_len as isize,
+      ref_len as isize,
+      params.terminal_bandwidth as isize,
+      params.excess_bandwidth as isize,
+      params.max_indel as isize,
+      params.allowed_mismatches as isize,
     )
```

Comment:
This probably comes from the base branch, but I only noticed it here. There's many arguments of the same type - easy to accidentally confuse their order. Unfortunately, Rust does not have named params, but we could consider:

 - to pass `params` directly, without referencing 4 of its fields before the call
 - if these fields and other args of the same type need to be passed separately, then perhaps it makes sense to make a struct (e.g. `CreateStripesParams`) and aggregate all or some of the functions' args there